diff --git a/Physics/src/ConvexHull.cpp b/Physics/src/ConvexHull.cpp
index 447e468..286749a 100644
--- a/Physics/src/ConvexHull.cpp
+++ b/Physics/src/ConvexHull.cpp
@@ -2419,7 +2419,7 @@ L2d750:
 		fcomp	st(1)		// 0x0002d76d
 		fnstsw	ax		// 0x0002d76f
 		test	ah, 0x41		// 0x0002d771
-		jne	L2d77e		// 0x0002d774
+		je	L2d77e		// 0x0002d774
 		fstp	st(0)		// 0x0002d776
 		mov	edi, ecx		// 0x0002d778
 		fld	dword ptr [esp + 0x78]		// 0x0002d77a
