diff --git a/Physics/src/ContactGeneration.cpp b/Physics/src/ContactGeneration.cpp
index 3a968a9..e299c36 100644
--- a/Physics/src/ContactGeneration.cpp
+++ b/Physics/src/ContactGeneration.cpp
@@ -3814,7 +3814,7 @@ L49110:
 		fld	dword ptr [ebp - 0x98]		// 0x00049134
 		fmul	dword ptr [ecx]		// 0x0004913a
 		faddp	st(1), st		// 0x0004913c
-		fadd	dword ptr [ebp - 0x68]		// 0x0004913e
+		fsub	dword ptr [ebp - 0x68]		// 0x0004913e
 		fcomp	dword ptr [ebp - 0x58]		// 0x00049141
 		fnstsw	ax		// 0x00049144
 		test	ah, 5		// 0x00049146
